op-node/flags/p2p_flags.go

```diff
@@ -34,15 +34,6 @@ var (
 		Value:    "none",
 		EnvVar:   p2pEnv("PEER_SCORING"),
 	}
-	PeerScoreBands = cli.StringFlag{
-		Name: "p2p.score.bands",
```

Comment:
We might want to leave a no-op hidden flag for compatibility, so this change can be rolled out more easily.

Reply:
Re-added as a hidden option.